syne_tune/blackbox_repository/conversion_scripts/scripts/nasbench201_import.py

```diff
@@ -240,7 +237,7 @@ def download_file_from_google_drive(id, destination):
 
                     session = requests.Session()
 
-                    params = {"id": id, "confirm": True}
+                    params = {"id": id, "confirm": "t"}
```

Comment:
Is this is typo or did the interface change?

Reply:
Yes it changed, but this does not fix it unfortunately, I left a comment instead.